packages/app-accounts/src/Account.tsx

```diff
@@ -329,90 +314,6 @@ function Account ({ address, className, filter, isFavorite, toggleFavorite }: Pr
           withExtended
         />
       </td>
-      <td className='number top'>
```

Comment:
Do we want to use `/** */` comment changes, in case someday someone wants to add it back? Also we only have two lines code differences in this PR?

Reply:
I thought, it will leave us with lots of commented code.. and send button works similar to 'Generic Transfer' functionality.. so we wanted to get rid of that..if anyone wants it, we can use it from the old code.. what do you say? cause this is like commenting 100 lines of code..

Reply:
Yeah would prefer to remove if unused and use history to find again